bluesky/utils.py

```diff
@@ -1071,13 +1071,13 @@ def apply_to_dict_recursively(d, f):
 class ProgressBar:
     def __init__(self, status_objs, delay_draw=0.2):
         """
-        Represent status objects with a progress bars.
+        Represent status objects with a text-based progress bars in a terminal.
```

Comment:
```
--- bars
+++ bar
```

Reply:
Oh, good catch. I decided to match the tenses the other way (status objects / progress bars) because multiple status objects plural can be represented at once on screen with stacked progress bars plural. Fix pushed.